### PR TITLE
Features/docu 576

### PR DESCRIPTION
**Pull request by benradstone**

### CODE REVIEW

File: docs/why-fusion/hive_replication.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
```
[the remark this answers is not in the document]
Comment (benradstone) [reply docs/why-fusion/hive_replication.md:1]: As discussed, currently writing up additional LiveAnalytics piece that will be added to this doc.

[the remark this answers is not in the document]
Comment (benradstone) [reply docs/why-fusion/hive_replication.md:1]: This has been added now, but I'm not 100% sure on the order I've put them in.

File: docs/why-fusion/hive_replication.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+id: hive_replication
+title: Hive Replication & Databricks integration
+sidebar_label: Hive Replication & Databricks integration
+---
+
```
Comment (catrionabarker) [inline docs/why-fusion/hive_replication.md:6]: I think a tiny intro on the relationship between LHV and DD might help set the scene before going into each one in detail. Something roughly along the lines of 

WANdisco offers 2 plugins which can be used together to implement a LiveAnalytics solution.
Fusion Plugin for Live Hive provides a method of replicating Hive metadata and the Fusion Plugin for Databricks Delta Lake is used with WANdisco Fusion to provide continuous replication from on-premises Hadoop analytics to Spark based cloud analytics with zero downtime and zero data loss.

File: docs/why-fusion/hive_replication.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
+id: hive_replication
+title: Hive Replication & Databricks integration
```
Comment (catrionabarker) [inline docs/why-fusion/hive_replication.md:3]: Integration (capital)

File: docs/why-fusion/hive_replication.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+The Fusion Plugin for Databricks Delta Lake works in conjunction with WANdisco Fusion and the Fusion Plugin for Live Hive to deliver WANdisco’s LiveAnalytics solution.
+
+When Hive tables are created in the source Hadoop zone, the Fusion Plugin for Databricks Delta Lake will create replicas of those tables in [Databricks](https://docs.databricks.com/getting-started/overview.html). Modify Hive schemas and see the same structure reflected in changes to matching Delta Lake tables.
+
+Data ingestion to Hive tables is also replicated to the Databricks cluster, which allows you to access the same information as Delta Lake content in a Databricks environment.
+
+## LiveAnalytics deployment overview
+
+A deployment will consist of two Zones:
```
Comment (catrionabarker) [inline docs/why-fusion/hive_replication.md:33]: A deployment consists of two zones:

File: docs/why-fusion/hive_replication.md (hunk 2 above)
Comment (benradstone) [reply docs/why-fusion/hive_replication.md:6]: I've added this in, plus a little more about using Live Hive as a standalone plugin in multiple zones. Let me know what you think.

**Pull request merged**